app/src/main/java/org/mozilla/fenix/tabstray/browser/AbstractBrowserTabViewHolder.kt

```diff
@@ -226,6 +237,47 @@ abstract class AbstractBrowserTabViewHolder(
                 false
             }
         }
+        // Since I immediately pass the event to onTouchEvent if it's not a move
+        // The ClickableViewAccessibility warning isn't useful
+        itemView.setOnTouchListener { view, motionEvent ->
+            when (motionEvent.actionMasked) {
+                MotionEvent.ACTION_DOWN -> {
+                    touchStartPoint = PointF(
+                        motionEvent.x,
+                        motionEvent.y
+                    )
+                }
+                MotionEvent.ACTION_UP, MotionEvent.ACTION_CANCEL -> {
+                    touchStartPoint = null
+                }
+                MotionEvent.ACTION_MOVE -> {
+                    val parent = itemView.parent as AbstractBrowserTrayList
+                    val touchStart = touchStartPoint
+                    val selected = holder.selectedItems
+                    val selectsOnlyThis = (selected.size == 1 && selected.contains(item))
+                    if (!parent.context.settings().searchTermTabGroupsAreEnabled &&
```

Comment:
Thank you @ssk97 for moving this forward and bearing with us! We would like to enable this feature in Nightly only for now to gather more feedback. Would you mind adding a `tabReorderingFeature` flag to `FeatureFlags.kt, set to `Config.channel.isNightlyOrDebug`, and checking if true here as well?

Reply:
Done. I had to separate out the function since the added condition hit the complexity limit of detekt.

Reply:
Thanks, @jonalmeida offered to take another look, and pending his review, we hope to land it in Nightly next week.